Color Picker/AppState.swift

```diff
@@ -128,6 +130,14 @@ final class AppState: ObservableObject {
 		}
 	}
 
+    @Published var hexColor = ""
+    @Published var hslColor = ""
+    @Published var rgbColor = ""
+    @Published var lchColor = ""
+    @Published var isPreventingUpdate = false
```

Comment:
Why are these moved to AppState? AppState is for state that is required globally. This state is only needed for picker screen. I think you can just pass `updateColorsFromPanel` to the `ColorPickerScreen` view.